modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue

```diff
@@ -109,12 +117,6 @@ export default {
     },
     moduleOptions() {
       return window.apos.adminBar;
-    },
-    unpublishedLabel() {
```

Comment:
You guys added this, so I assume you decided you don't need it anymore.

Reply:
Right it now comes from `moduleOptions.contextLabels`